src/Repositories/DiscountRepository.php

```diff
@@ -117,8 +120,17 @@ public function getActiveDiscounts()
             )
             ->setParameter('active', true);
 
-        return $qb->getQuery()
+        $results = $qb->getQuery()
```

Comment:
Not sure about adding the discount like this.
I didn't want to add it in the database since it should always be here.
Do you think this is okay?